smarts/core/bubble_manager.py

```diff
@@ -76,6 +77,13 @@ class Bubble:
     def __init__(self, bubble: SSBubble, road_map: RoadMap):
         geometry = bubble.zone.to_geometry(road_map)
 
+        bounding = geometry.bounds
+        self.radius = math.sqrt(
+            (bounding[2] - bounding[0]) * (bounding[2] - bounding[0])
+            + (bounding[3] - bounding[1]) * (bounding[3] - bounding[1])
+        )
```

Comment:
This does not seem quite right... 

`pow(maxx - minxx, 2) + pow(maxy - miny, 2)`

I think this ends up significantly larger than it should be. I think it should be something like `pow(maxx - centroidx, 2) + pow(centroidy - miny, 2) + 0.1` perhaps?